mmf/utils/build.py

```diff
@@ -10,10 +10,15 @@
 from mmf.common.registry import registry
 from mmf.datasets.processors.processors import Processor
 from mmf.utils.configuration import Configuration
-from mmf.utils.distributed import is_dist_initialized
+from mmf.utils.distributed import is_dist_initialized, is_xla, is_master, synchronize
 from mmf.utils.general import get_optimizer_parameters
 from omegaconf import DictConfig, OmegaConf
 
+try:
+    import torch_xla.core.xla_model as xm
```

Comment:
Does `torch_xla` come with any stringent dependencies that would cause MMF installation to fail in certain scenarios? If not maybe, we can just add it to requirements and assume xla would be there.

Though, let's take a look at this later, I will also need to check what's the status of xla in fb internal.

Reply:
This is pattern used in fairseq, There are some discussion to make this seemless. torch-xla does not yet have a pypy package, therefore adding to requirement.txt can cause friction for some users.